autoload/lsp/ui/vim.vim

```diff
@@ -3,7 +3,7 @@ let s:last_req_id = 0
 function! lsp#ui#vim#definition() abort
     let l:servers = filter(lsp#get_whitelisted_servers(), 'lsp#capabilities#has_definition_provider(v:val)')
     let s:last_req_id = s:last_req_id + 1
-
+    let l:ctx = { 'counter': len(l:servers), 'list':[] }
```

Comment:
I would prefer to pass the entire servers (`{ 'servers': l:servers, .... }`) in `ctx` and then check the length of it in `s:handle_location` instead of passing `jump_if_one`.